core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -163,6 +165,17 @@
       "Leave blank to use Jetty's default.";
   protected static final String SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_DEFAULT = "";
 
+  public static final String AUTHENTICATION_METHOD_CONFIG = "authentication.method";
+  public static final String AUTHENTICATION_METHOD_NONE = "NONE";
+  public static final String AUTHENTICATION_METHOD_BASIC = "BASIC";
+  public static final String AUTHENTICATION_METHOD_DOC = "Method of authentication. Must be BASIC to enable authentication. You must supply a valid JVM config file for java.security.auth.login.config for the appropriate authentication provider.";
```

Comment:
Instead of JVM config, I think I would say JAAS config. And mention that `java.security.auth.login.config` is a system property.